app/controllers/api/v0/reports_controller.rb

```diff
@@ -10,7 +10,8 @@ class ReportsController < Api::V0::BaseController
       before_action :validate_report, :authorize_report, :validate_query
 
       def show
-        @report = report_class.new(current_api_user, ransack_params, report_options)
+        params[:report_format] = 'json'
```

Comment:
Why do we restrict to JSON? For some applications it can be very handy to get a CSV.
```suggestion
        params[:report_format] ||= 'json'
```

?

Reply:
you mean some applications will automate the download of the file and then process it? if we want to handle CSV, we shall also replace th way `Api::ReportController` render the report, cause for now it's `render json: { data: @report.to_csv }`

maybe we can first having this JSON api works, and let see if there is one day a use case for CSV? or does the use case already exists?

Reply:
Good point about the rendering format. I'm not 100% sure but I guessed that there's already a use case for CSV. But you are right, lets wait for someone to ask and then we can do it properly.